### PR TITLE
Check markdown links validity in CI

### PR DESCRIPTION
**Pull request by rmalmain**

### CODE REVIEW

**Comment by domenukk** [comment]
Ooops didn't mean to merge yet, sorry

**Comment by domenukk** [comment]
We can open a new PR though to fix those, I don't think un-fixing the current ones makes sense

**Comment by rmalmain** [comment]
no worries it's fine. we can continue in another pr if it's still broken

File: docs/src/design/migration-0.11.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Some cross-platform things in bolts include
 * ShMem: A cross-platform (Windows, Linux, Android, MacOS) shared memory implementation
 * LLMP: A fast, lock-free IPC mechanism via SharedMap
 * Core_affinity: A maintained version of `core_affinity` that can be used to get core information and bind processes to cores
-* Rands: Fast random number generators for fuzzing (like [RomuRand](http://www.romu-random.org/))
+* Rands: Fast random number generators for fuzzing (like [RomuRand](https://www.romu-random.org/))
```
Comment (domenukk) [inline docs/src/design/migration-0.11.md:15]: TLS is broken on this domain

**Comment by domenukk** [comment]
Appears to still be broken

**Comment by rmalmain** [comment]
we can continue here #2497

File: docs/src/design/migration-0.11.md (hunk 1 above)
Comment (rmalmain) [reply docs/src/design/migration-0.11.md:15]: both are broken according to the action, not sure what we should do with this

**Pull request merged**